tests/integration/aof.tcl

```diff
@@ -511,23 +511,15 @@ tags {"aof external:skip"} {
         }
     }
 
-    test {EVAL can't processes writes in AOF when doing "debug loadaof" in read-only slaves} {
-        start_server [list overrides [list dir $server_path appendonly no aof-use-rdb-preamble no]] {
-            r config set appendonly yes
-            waitForBgrewriteaof r
-            create_aof {
-                append_to_aof [formatCommand select 9]
-                append_to_aof [formatCommand eval {redis.call("set",KEYS[1],"100")} 1 foo]
-                append_to_aof [formatCommand eval {redis.call("incr",KEYS[1])} 1 foo]
-                append_to_aof [formatCommand eval {redis.call("incr",KEYS[1])} 1 foo]
-            }
-            r config set slave-read-only yes
-            r slaveof 127.0.0.1 0
-            r debug loadaof
-            after 100
-            r slaveof no one
+    test {EVAL can't process writes from AOF in read-only replicas} {
+        create_aof {
+            append_to_aof [formatCommand select 9]
+            append_to_aof [formatCommand eval {redis.call("set",KEYS[1],"100")} 1 foo]
+            append_to_aof [formatCommand eval {redis.call("incr",KEYS[1])} 1 foo]
+            append_to_aof [formatCommand eval {redis.call("incr",KEYS[1])} 1 foo]
+        }
+        start_server [list overrides [list dir $server_path appendonly yes aof-use-rdb-preamble no replica-read-only yes replicaof "1234 1234"]] {
```

Comment:
```suggestion
        start_server [list overrides [list dir $server_path appendonly yes replica-read-only yes replicaof "127.0.0.1 0"]] {
```

Reply:
Oops, looks like I forgot to clean the messes up

Reply:
why delete `aof-use-rdb-preamble no`? Default it is `yes`.

Reply:
the config was only needed because we used BGREWRITEAOF to generate an AOF (after running EVAL commands).
but now that we generate the config manually, this config is meaningless (doesn't affect the test).